elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -322,17 +324,18 @@ private FilterExpression equalityExpression(String argument, Path path, List<Obj
          * @return Returns Predicate for '=isnull=' case depending on its arguments.
          */
         private FilterExpression buildIsNullOperator(Path path, List<String> arguments) {
-            Operator elideOP;
+            String arg = arguments.get(0);
```

Comment:
Is it required that there is an argument here?  This will NPE if no argument is provided.